### PR TITLE
V2 add time manipulation

### PR DESCRIPTION
**Pull request by jeffyjin**
Hi Lazee, as per your request, I re-applied the PR to V2 and added test cases and updated README.md. Please let me know your thoughts. Thanks

### CODE REVIEW

**Comment by lazee** [comment]
Wow! Thanks! I will go through it within a couple of days @jeffyjin

File: src/main/java/no/api/freemarker/java8/EnhancedJava8ObjectWrapper.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import no.api.freemarker.java8.time.TemporalDialerAdapter;
+import no.api.freemarker.java8.zone.ZonedDateTimeStrategy;
+
+import java.time.temporal.Temporal;
+
+/**
+ * Freemarker ObjectWrapper that extends the Java8ObjectWrapper with support for time manipulations in all temporal classes in the new java.time api.
+ * Such time manipulation operations include plusSeconds, plusMinutes, etc.
+ */
+public class EnhancedJava8ObjectWrapper extends Java8ObjectWrapper {
```
Comment (lazee) [inline src/main/java/no/api/freemarker/java8/EnhancedJava8ObjectWrapper.java]: Why this Enhanced wrapper ? Couldn't we just add it to the Java8ObjetWrapper ?

**Review by lazee — COMMENTED** [review]
Except from my question about the enhanced wrapper, this looks very good and is beautifully done.

File: src/main/java/no/api/freemarker/java8/EnhancedJava8ObjectWrapper.java (hunk 1 above)
Comment (jeffyjin) [reply src/main/java/no/api/freemarker/java8/EnhancedJava8ObjectWrapper.java]: Of course. I didn't want to be too intrusive. I actually more prefer to merge the changes into Java8ObjetWrapper. Will modify PR soon

Comment (jeffyjin) [reply src/main/java/no/api/freemarker/java8/EnhancedJava8ObjectWrapper.java]: Done. Please review

Comment (lazee) [reply src/main/java/no/api/freemarker/java8/EnhancedJava8ObjectWrapper.java]: Thanks! I will look into this and merge it soon. I think v2 should be released within a week or so.

**Comment by lazee** [comment]
2.0.0 release @jeffyjin. Available tomorrow from central repo. Again, thank you!

**Pull request merged**